uber/decorators.py

```diff
@@ -279,6 +279,12 @@ def render(template_name_list, data=None):
     return rendered.encode('utf-8')
 
 
+def render_empty(template_name_list):
+    env = JinjaEnv.env()
+    template = env.get_or_select_template(template_name_list)
+    return str(open(template.filename, 'r').read())
```

Comment:
This is probably fine. It's basically equivalent to how Jinja opens and reads files: https://github.com/pallets/jinja/blob/master/jinja2/utils.py#L149

According to the [python docs](https://docs.python.org/2/howto/unicode.html) A more robust way of reading utf-8 would be:
```python
import codecs
codecs.open(template.filename, encoding='utf-8', mode='r').read()
```